classes/podsfield_pandarepeaterfield.php

```diff
@@ -375,6 +395,34 @@ public function input ( $name, $value = null, $options = null, $pod = null, $id
 				if( self::$tbs_arr['pod_' . $savedtb_int ]['type'] == 'post_type' ){
 					$join_str = 'INNER JOIN  `' . $table_prefix . 'posts` AS post_tb ON post_tb.ID = main_tb.id';
 				}
+
+				// order
+				$order_str		=	'CAST( `pandarf_order` AS UNSIGNED ) ';
+				$orderInfo_str	=	'Ordered by ';
```

Comment:
The way you are building `$orderInfo_str` is not translatable. Consider using an array to catch the ordered fields and then use `$orderInfo_str = sprintf( __( 'Ordered by %s', 'your-text-domain' ), implode( ', ', $ordered_fields ) )`